numpy/core/fromnumeric.py

```diff
@@ -3284,6 +3286,9 @@ def mean(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
         sub-class' method does not implement `keepdims` any
         exceptions will be raised.
 
+    where : array_like of bool, optional
+        Elements to include in the mean. See `~numpy.ufunc.reduce` for details.
+
```

Comment:
Does this have to be separated with a `.. versionadded:: <whatever>`?

Reply:
That should be as the last line of this description

Reply:
Alright, updated it to `1.19.0`, or does it have to be the next, so `1.19.1`?

Reply:
1.19.0 is fine